plugins/star-rating/api/api.js

```diff
@@ -188,9 +188,11 @@ var plugin = {},
                                     return false;
                                 }
                             });
-                    }
-                }
-                return true;
+                    } else return true;
+                } else {
+                    console.log(currEvent.key);
+                    return true;
+                } 
             })
```

Comment:
I think previous way of returning always true, was more understandable, was there any fault to it?
At least let's remove console.log